cityofaustin/atd-moped
# Updates signal dashboard to use new project personnel schema

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10766

## Testing
**URL to test:**  [Deploy preview](https://deploy-preview-874--atd-moped-main.netlify.app/moped/session/signin)

**Steps to test:**
1. Navigate to the [signal dashboard](https://deploy-preview-874--atd-moped-main.netlify.app/moped/views/signal-projects).
2. Observe that the page loads and there are users in the "Designer" and "Inspector columns. 
3. Add two team members to a project - set one as a designer and one as an inspector
4. Find your project in the signal dashboard table. Verify the "Designer" and "Inspector" columns populate correctly.

---
#### Ship list
